.gitignore

```diff
@@ -59,3 +59,5 @@ pnpm-debug.log
 # Package names containing dots may match any of the above.
 # Just allow any directory within the types directory directly.
 !/types/*/
+
+docs
```

Comment:
We do not take doc files so please dont add them, they wont be part of final package